### PR TITLE
fix: display project alias instead of name

### PR DESCRIPTION
**Pull request by cd1989**
**What this PR does / why we need it**:

Display project alias instead of name.

image

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @supereagle 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: web/src/components/project/component/List.js [hunk 1]
```diff
@@ -44,7 +40,7 @@ class List extends React.Component {
     const columns = [
       {
         title: intl.get('name'),
-        dataIndex: 'metadata.name',
+        dataIndex: 'metadata.annotations["cyclone.io/alias"]',
```
Comment (huxiangtao) [inline web/src/components/project/component/List.js]: if get undefined , loadash more better?

Comment (cd1989) [reply web/src/components/project/component/List.js]: Here `metadata.annotations["cyclone.io/alias"]` is key path in the object, not retrieve data here.

**Pull request merged**